cypress/page/iframe.page.ts

```diff
@@ -0,0 +1,34 @@
+class IFramePage {
+
+    private baseUrl: string
+    private cssUrl: string
+    private selectorIFrame: string
+    private selectorIFrameTitle: string
+    private selectorIFrameCssButton: string
+
+    constructor() {
+        this.baseUrl = "https://www.w3schools.com/html/html_iframe.asp"
+        this.cssUrl = "https://www.w3schools.com/css/default.asp"
+        this.selectorIFrame = "#main > div > iframe[title='W3Schools HTML Tutorial']"
```

Comment:
I believe this selector is enough
```suggestion
      iframe[title='W3Schools HTML Tutorial']"
```